.eslintrc.js

```diff
@@ -13,5 +13,6 @@ module.exports = {
     'no-return-await': 'off',
     'simple-import-sort/imports': 'warn',
     'simple-import-sort/exports': 'warn',
+    'dot-notation': 'off',
```

Comment:
Why turn off dot notation? What value does it provide? 🤔

Reply:
The value it provides is ESLint and TypeScript agreeing on what's valid syntax:

https://github.com/sanity-io/groq-js/assets/81981/18195ee0-9c63-4b3a-a091-c625431af679